README.md

```diff
@@ -1,3 +1,94 @@
 # promises-debugging
 
-Placeholder
+Promises have been available to Node.js developers for several years in various
+forms (as a built-in feature of the language or as npm modules). However,
+debugging errors created from promises tasks can still present some challenges
+in some use cases.
+
+As a result, an _informal_ working group was formed with the goal to make the
+experience of Node.js developers less challenging when debugging such errors.
+
+## Goals
+
+The goals of this working group are:
+
+- Identifying, investigating and proposing solutions for gaps in promises error
+  debugging use cases.
+
+- Documenting best practices about debugging promises errors.
+
+## Scope
+
+Promises debugging and error handling can cover a large surface area of the
+Node.js runtime. We want to make sure this working group is focused on just the
+critical parts of that surface area so that we can make concrete progress as
+quickly as possible.
+
+Other individuals or groups can tackle the challenges that are out of scope for
+this group.
+
+### In scope
+
+- Ability to use post-mortem debugging techniques (using both OS level core
+  dumps and other forms of artifacts) to debug unhandled errors created by
+  promises tasks.
+
+- Ability to use live debugging techniques to debug unhandled errors created by
+  promises tasks.
+
+### Out of scope
+
+Any aspect of debugging promises that is not critical to _enabling_ debugging
+use cases is not in scope. For instance:
+
+- the performance of promises when using live debuggers is not relevant for this
+  working group, even if it is very important for Node.js users.
+
+- the user experience of using promises, even when it relates to providing a
+  safer API, is not relevant to this working group. For instance, whether
+  resolving or rejecting a promise that has already been resolved triggers an
+  error is not relevant to this working group.
```

Comment:
Not sure why this would be out of scope either.

Reply:
Does my answer to @benjamingr's comment above help?